frontend/src/views/ensemble/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from "react";
+import "./index.css";
+
+const Ensemble: FC = () => {
```

Comment:
This is going to collide with the type in common, `Ensemble`. You should name this `EnsembleView` or something similar.